packages/clients/src/api/baremetal/v1/index.ts

```diff
@@ -1,5 +1,5 @@
 export { BaremetalV1UtilsAPI as API } from './api.utils'
-export { BaremetalPrivateNetworkV1GenAPI as PrivateNetworkAPI } from './api.gen'
```

Comment:
this file should be removed as the others products or we don't have export inside baremetal/index  ? @Vinzius

Reply:
Baremetal has manual utils, so we have to keep manual index :(